systems/api/api-gateway/pkg/config.go

```diff
@@ -0,0 +1,77 @@
+package pkg
+
+import (
+	"time"
+
+	"github.com/gin-contrib/cors"
+	"github.com/ukama/ukama/systems/common/config"
+	"github.com/ukama/ukama/systems/common/rest"
+)
+
+type Config struct {
+	config.BaseConfig `mapstructure:",squash"`
+	DB                *config.Database `default:"{}"`
+	Server            rest.HttpConfig
+	// Services          GrpcEndpoints  `mapstructure:"services"`
+	// Service          *uconf.Service
```

Comment:
Some commented code in this file, Let's remove it if we don't need otherwise we can add TODO for ref.

Reply:
Addressed.